src/Test.UnitTests.Sarif.Driver/DriverExtensionMethodsTests.cs

```diff
@@ -242,28 +242,53 @@ public void ValidatingMultipleFilesOutputOptions_ProducesExpectedResults()
         [Fact]
         public void ValidateAnalyzeOutputOptions_ProducesExpectedResults()
         {
-            AnalyzeOptionsBase analyzeOptionsBase = new TestAnalyzeOptions();
-            TestAnalysisContext context = new TestAnalysisContext();
+            var context = new TestAnalysisContext();
+            var analyzeOptionsBase = new TestAnalyzeOptions();
 
-            //  quiet false, output empty
+            // quiet false, output empty
             analyzeOptionsBase.Quiet = false;
             analyzeOptionsBase.OutputFilePath = null;
             Assert.True(analyzeOptionsBase.ValidateOutputOptions(context));
 
-            //  quiet false, output non-empty
+            // quiet false, output non-empty
             analyzeOptionsBase.Quiet = false;
             analyzeOptionsBase.OutputFilePath = "doodle";
             Assert.True(analyzeOptionsBase.ValidateOutputOptions(context));
 
-            //  quiet true, output empty
+            // quiet true, output empty
             analyzeOptionsBase.Quiet = true;
             analyzeOptionsBase.OutputFilePath = null;
             Assert.False(analyzeOptionsBase.ValidateOutputOptions(context));
 
-            //  quiet true, output non-empty
+            // quiet true, output non-empty
             analyzeOptionsBase.Quiet = true;
             analyzeOptionsBase.OutputFilePath = "doodle";
             Assert.True(analyzeOptionsBase.ValidateOutputOptions(context));
+
+            // postUri non-empty and outputFilePath empty = false
+            analyzeOptionsBase.PostUri = "doodle";
```

Comment:
>doodle [](http://example.com/codeflow?start=42&length=6)

Why 'doodle'? You are repeating someone else's pattern but you should always ask why. 

`doodle` doesn't mean anything here, that was James Winkler's preferred term for meaningless data. In this case, you can actually use something descriptive, like '[https://NotNull.example.com](https://NotNull.example.com)'

For a file use, 'SomeFile.txt' or whatever.

These terms manage to convey something about what's going on.